pantheon/src/test/java/tech/pegasys/pantheon/cli/PantheonCommandTest.java

```diff
@@ -328,14 +336,36 @@ public void NoOverrideDefaultValuesIfKeyIsNotPresentInConfigFile() throws IOExce
     assertThat(commandErrorOutput.toString()).isEmpty();
   }
 
+  @Test
+  public void nodekeyOptionMustBeUsed() throws Exception {
+    final File file = new File("./specific/key");
+
+    parseCommand("--node-private-key", file.getPath());
+
+    verify(mockControllerBuilder)
+        .build(
+            any(),
+            isNotNull(),
+            any(),
+            eq(false),
+            any(),
+            anyBoolean(),
+            fileArgumentCaptor.capture());
+
+    assertThat(fileArgumentCaptor.getValue().compareTo(file)).isZero();
```

Comment:
This is a particularly unusual form of assertion and won't provide very useful feedback.  Is `assertThat(fileArgumentCaptor.getValue()).isEqualTo(file)` not sufficient?  If not maybe you could just compare the absolute file paths?

Reply:
Will try that, thanks.